src/TogglePassword/assets/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "@symfony/ux-toggle-password",
+    "description": "Toggle visibility of password inputs for Symfony Forms",
+    "license": "MIT",
+    "version": "1.0.0",
+    "main": "dist/controller.js",
+    "types": "dist/controller.d.ts",
+    "config": {
+        "css_source": "src/style.css"
+    },
```

Comment:
You can remove this `config` key. I had never noticed that we had this on a few other, old packages. I believe it was added by accident there.

Reply:
sure, i'll remove it 👍🏼 
I thought it was needed for the css file provided with the bundle !

Reply:
It probably was in a very early WIP version of UX + Flex - that's my best guess :)